### PR TITLE
Add deprecation warnings when addons are required via main package

### PR DESCRIPTION
**Pull request by shilman**
Issue: #1017

## What I did

Use `action` and `linkTo` addons just like any other storybook addon.
Update examples, docs, and add deprecation warning to code.

## How to test
First run:
```
npm run bootstrap
cd examples/cra-storybook
npm run storybook
```
Open browser console, and there should be no deprecation warnings.
Then, update the stories to use:

```js
import { storiesOf, action, linkTo from } from '@kadira/storybook'
```
Deprecation warnings should appear in the browser console.

### CODE REVIEW

File: packages/addon-info/.storybook/addons.js [hunk 1]
```diff
@@ -1,2 +1,2 @@
-import '@kadira/storybook/addons';
+import '@kadira/storybook-addon-actions/register';
```
Comment (ndelangen) [inline packages/addon-info/.storybook/addons.js]: Is this missing addon-links?
Or is this on purpose?

File: packages/react-storybook/src/client/index.js [hunk 2]
```diff
@@ -7,5 +8,15 @@ export const configure = previewApi.configure;
 export const getStorybook = previewApi.getStorybook;
 
 // NOTE export these to keep backwards compatibility
-export { action } from '@kadira/storybook-addon-actions';
-export { linkTo } from '@kadira/storybook-addon-links';
+import { action as _action } from '@kadira/storybook-addon-actions';
+import { linkTo as _linkTo } from '@kadira/storybook-addon-links';
+
+export const action = deprecate(
+  _action,
```
Comment (ndelangen) [inline packages/react-storybook/src/client/index.js]: This dangling underscore will become an eslint error in the future, can you pick another naming pattern?

**Review by ndelangen — CHANGES_REQUESTED** [review]

File: packages/react-storybook/src/client/index.js (hunk 2 above)
Comment (shilman) [inline packages/react-storybook/src/client/index.js]: fixed!

File: packages/addon-info/.storybook/addons.js (hunk 1 above)
Comment (shilman) [inline packages/addon-info/.storybook/addons.js]: on purpose. links are not used in this example.

**Review by ndelangen — APPROVED** [review]

**Pull request merged**